### ISSUE TITLE
Fine-tuning QuartzNet 15*5 with multiple speaker audio dataset

### ISSUE THREAD

**Issue by MelikaBahmanabadi** [NVIDIA/NeMo#2375]
Hi,
Can I use Persian Multiple speaker audio datasets for fine-tuning QuartzNet15*5? Will be the WER low as I fine-tune it with a single speaker dataset?
Thanks

**Environment overview**

- Environment location: Google Colab
- Method of NeMo install: !pip install nemo_toolkit[asr]
- NeMo version: 1.0.0
- Learning Rate: 1e-3

**Environment details**

- OS version : "Ubuntu20.04.3 LTS"
- PyTorch version : "1.7.1"

**Comment by titu1994** [comment]
The model might train fine, but I'm evaluation on other speakers will have poor WER. It can be tried, but for proper generalization more data from multiple speakers would be useful
